spec/controllers/api/irs_attempts_api_controller_spec.rb

```diff
@@ -31,6 +31,21 @@
   let(:existing_event_jtis) { existing_events.map(&:first) }
 
   describe '#create' do
+    context 'with CSRF protection enabled' do
+      around do |ex|
+        ActionController::Base.allow_forgery_protection = true
+        ex.run
+        ActionController::Base.allow_forgery_protection = false
+      end
```

Comment:
This pattern is used in a few places now but never _quite_ the same way. Some day this might make sense as a shared example to pull in, but I think it would just complicate things I tried to force it here.